cmd/notary/keys.go

```diff
@@ -268,27 +268,12 @@ func (k *keyCommander) keysExport(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	// Search for this key in all of our keystores, determine whether this key has a GUN
-	keyGun := ""
-	keyRole := ""
-	for _, store := range ks {
-		for keypath, role := range store.ListKeys() {
-			if filepath.Base(keypath) == keyID {
-				keyRole = role
-				if role == data.CanonicalRootRole {
-					continue
-				}
-				dirPath := filepath.Dir(keypath)
-				if dirPath != "." { // no gun
-					keyGun = dirPath
-				}
-				break
-			}
-		}
+	cs := cryptoservice.NewCryptoService("", ks...)
```

Comment:
Yeah we would need to include a gun for both `CryptoService.Create` and `CryptoService.AddKey`.  I'm leaning toward making this change in another PR, since it will affect a lot of our repo code?